common/common-rest/src/main/java/io/servicecomb/common/rest/RestProducerInvocation.java

```diff
@@ -64,6 +65,18 @@ public void invoke(Transport transport, HttpServletRequestEx requestEx, HttpServ
 
   protected void scheduleInvocation() {
     OperationMeta operationMeta = restOperationMeta.getOperationMeta();
+    try {
+      Object[] args = RestCodec.restToArgs(requestEx, restOperationMeta);
```

Comment:
this moved decode from executor to netThread, not so good.
prepareStartTimeMetrics only care about time and qualifiedName, qualifiedName can be got from operationMeta
prepareStartTimeMetrics can return a metrics object, and then save it to invocation in runOnExecutor.

Reply:
fixed